Tests/js/views/services/assets/ez-locationviewviewservice-tests.js

```diff
@@ -478,9 +479,316 @@ YUI.add('ez-locationviewviewservice-tests', function (Y) {
+                run: function (options, callback){
+                    callback(true);
+                }
+            });
+
+            this.service.on('confirmBoxOpen', function (e) {
+                e.config.confirmHandler.apply(this);
+            });
+
+            this.service.on('notify', function (e) {
+                this.once('notify', function (e) {
+                    notified = true;
+
+                    Assert.isObject(e.notification, "The event facade should provide a notification config");
+                    Assert.areEqual(
+                        "error", e.notification.state,
+                        "The notification state should be 'error'"
+                    );
+                    Assert.isString(
+                        e.notification.text,
+                        "The notification text should be a string"
+                    );
+                    Assert.isTrue(
+                        e.notification.text.indexOf(contentName) !== -1,
+                        "The notification text should contain content name"
+                    );
+                    Assert.isTrue(
+                        e.notification.identifier.indexOf(locationId) !== -1,
+                        "The notification identifier should contain location id"
+                    );
+                    Assert.areSame(
+                        0, e.notification.timeout,
+                        "The notification timeout should be set to 0"
+                    );
+                });
+            });
+
+            this.service.fire('whatever:sendToTrashAction', {content: content});
+            Assert.isTrue(notified, "The notified event should have been fired");
+        },
+
+        "Should notify user about success of sending content to trash": function () {
+            var content = new Mock(),
+                parentLocation = new Mock(),
+                locationId = 'raul-gonzalez-blanco',
+                contentName = 'pierlugi-collina',
+                notified = false;
+
+            this.service.set('path', [{location: parentLocation}]);
+
+            Mock.expect(this.location, {
+                method: 'get',
+                args: ['id'],
+                returns: locationId,
+            });
+
+            Mock.expect(content, {
+                method: 'get',
+                args: ['name'],
+                returns: contentName,
+            });
+
+            Mock.expect(parentLocation, {
+                method: 'get',
+                args: ['id'],
+                returns: 'alan-shearer'
+            });
+
+            Mock.expect(this.location, {
+                method: 'trash',
+                args: [Mock.Value.Object, Mock.Value.Function],
+                run: function (options, callback){
+                    callback(false);
+                }
+            });
+
+            Mock.expect(this.app, {
+                method: 'navigateTo',
+                args: [Mock.Value.String, Mock.Value.Object]
```

Comment:
here as well, the parameter are important, you should put the first to `'viewLocation`' and add a `run` function in the expectation to check that the second parameter contains the expected property.